src/Map.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Tile from './Tile';
+import './Map.css';
+
+function Map() {
+    return (
+      <table className="Map">
+        <tbody>
+          <tr><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /></tr>
```

Comment:
Surely we can use a for loop here :P

Reply:
Haha, my embarrassing interim state exposed! Yes, we can :P

Reply:
I think my idea for where to go from here may actually have been to map over an array of tiles...? What are your thoughts for how we're going to store map state?